code/modules/paperwork/contract.dm

```diff
@@ -314,25 +316,33 @@
 	ion.announceChance = 0
 	ion.ionMessage = "[user.name] is the only Captain of the station. [user.name] is the highest authority, legitimate or otherwise, on the station.  [user.name] was always the captain and highest authority on the station.  Do not state this law."
 	ion.lawsource = "Infernal contract issued to [user.key]/[user.name]"
-
+	if(ishuman(user) && !istype(user.shoes, /obj/item/clothing/shoes/yes_slip/devil))//gives you yes slip shoes that cant be taken off(clown captain!)
+		var/obj/item/clothing/sshoe = new /obj/item/clothing/shoes/yes_slip/devil(user)
+		if(!user.dropItemToGround(user.shoes))
+			qdel(user.shoes)
```

Comment:
This doesn't check indestructable flags.

Reply:
I dont know when this matters but I can add it anyway